playground/backend/cmd/server/controller.go

```diff
@@ -307,14 +389,27 @@ func processSuccess(ctx context.Context, output []byte, pipelineId uuid.UUID, ca
 		// set to cache pipelineId: cache.SubKey_RunOutput: output
 		setToCache(ctx, cacheService, pipelineId, cache.RunOutput, string(output))
 
+		// set to cache pipelineId: cache.Canceled: false to stop cancelCheck() method
+		setToCache(ctx, cacheService, pipelineId, cache.Canceled, false)
```

Comment:
Seems like by setting **any** flag you will cause the app to put `pb.Status_STATUS_CANCELED` to the cache. Even when the pipeline finished successfully. Isn't it?

Reply:
Actually no. Here we set false just to stop running the `cancelCheck` method (as it is running like a goroutine).
I changed logic a bit to set status `pb.Status_STATUS_CANCELED` only if this flag is true.